.github/workflows/test.yml

```diff
@@ -30,14 +30,15 @@ jobs:
         with:
           go-version: ${{ matrix.go-version }}
           cache: true
-
-      - name: go test
+      - uses: autero1/action-gotestsum@v2.0.0
```

Comment:
are you sure about this source? I don't like using small custom actions (based on the name)

Reply:
Especialy just to avoid a go get - you could also cache it instead